plugin-server/tests/main/process-event.test.ts

```diff
@@ -1273,6 +1273,181 @@ test('distinct with anonymous_id which was already created', async () => {
     expect(person.is_identified).toEqual(true)
 })
 
+test('distinct with anonymous_id which was already created triggers foreign key updates', async () => {
```

Comment:
Nitpick (probably too late to move this now) but this test file is soft-deprecated now.

plugin-server/tests/worker/ingestion/event-pipeline/event-pipeline-integration.test.ts is a better spot for this.

Reply:
Actually these are more like unit tests rather than integration ones it seems.

In this case - separate `describe` blocks in `person-state.ts` would be preferred.

Reply:
Sure thing